man/ff-deps.1.md

```diff
@@ -0,0 +1,135 @@
+% FF-DEPS(1)
+% Daniel de Kok, Sebastian Pütz
+% Apr 6, 2019
+
+NAME
+====
+
+**ff-deps** -- train dependency-based word embeddings with subword representations
```

Comment:
Would it be possible to shorten this a bit so that it fits a standard 80 columns terminal? (At least, I think iTerm2 uses 80 cols by default and this line is wrapped.)

Reply:
Terminal on ubuntu also defaults to 80 cols. I added a setting to my .vim_rc so content that goes beyond 80 chars gets marked.

Reply:
Indeed, that's very handy. I have the following in my `.vimrc`, which puts a nice vertical ruler to indicate a boundary:

```
set textwidth=76
set colorcolumn=+1
```